packages/react/lib/runtime.js

```diff
@@ -0,0 +1,16 @@
+const deprecate = require('depd')('hops-react');
+
+deprecate(
+  '[DEP004] Do not use deep imports to "hops-react" (https://github.com/xing/hops/blob/master/DEPRECATIONS.md#dep004).'
```

Comment:
The URL to the `DEPRECATIONS.md` should be relative.

Reply:
Why should it be relative? It is a URL that will be printed on the CLI, so that users can click on it. A relative URL would not work there.

Reply:
Sorry, got the context wrong.